src/build.py

```diff
@@ -875,6 +901,7 @@ def Emscripten(use_asm=True):
   def WriteEmscriptenConfig(infile, outfile):
     with open(infile) as config:
       text = config.read().replace('{{WASM_INSTALL}}', INSTALL_DIR)
```

Comment:
I dunno, your version looks like more line noise to me... :) (and more lines :D)